platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -252,6 +268,94 @@ Unresolved content resize issues are documented in issue tickets and listed as k
 *   [F80: Failure of Success Criterion 1.4.4 when text-based form controls do not resize when visually rendered text is resized up to 200%](http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F80)
 
 
+### Pointer Gestures
+
+**Who:** Frontend engineer
+
+**When:** As part of daily development
+
+#### Steps to test
+
+*   Use a mobile device to test a functionality of feature
+    *   Make sure you can fully utilize feature using only 1 finger to gesture (tapping, swiping, etc)
+
+#### Definition of done
+
+All functionality is available when limited to 1 finger to gesture
+
+#### Related resources
+
+*   [2.5.1 Pointer Gestures](https://www.w3.org/TR/WCAG21/#pointer-gestures)
+*   [Understanding Success Criterion 2.5.1: Pointer Gestures](https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html)
+
+
+### Pointer Cancellation
+
+**Who:** Frontend engineer
+
+**When:** As part of daily development
+
+#### Steps to test
+
+*   Touch/click down on a button/link/feature
+    *   The button/link/feature should not trigger
+*   Untouch/click up on a button/link/feature
+    *   The button/link/feature should trigger
```

Comment:
1. Touch/click down on a button/link/feature. The button/link/feature should not trigger.
 2. Untouch/click up on a button/link/feature. The button/link/feature should trigger.

(I feel like there's probably a cleaner way to describe this, but I fear I will jump headlong into a rabbit hole of how describe mouse and tap interactions and overthink things 😄   The revision I'm proposing is based on our writing guidelines that a bullet should always have more than 1 sub-bullet and that we should use numbered list for instructions describing a series of sequential steps.)